x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/moving_average.tsx

```diff
@@ -168,3 +169,67 @@ function MovingAverageParamEditor({
     </EuiFormRow>
   );
 }
+
+const MovingAveragePopup = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
+          {i18n.translate('xpack.lens.indexPattern.movingAverage.helpText', {
+            defaultMessage: 'How does it work?',
+          })}
+        </HelpPopoverButton>
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      isOpen={isPopoverOpen}
+      title={i18n.translate('xpack.lens.indexPattern.movingAverage.titleHelp', {
+        defaultMessage: 'How does moving average work?',
+      })}
+    >
+      <p>
+        <FormattedMessage
+          id="xpack.lens.indexPattern.movingAverage.basicExplanation"
+          defaultMessage="The Moving Average slides a window across the data and emits the average value of that window."
+        />
+      </p>
+
+      <p>
+        <FormattedMessage
+          id="xpack.lens.indexPattern.movingAverage.longerExplanation"
+          defaultMessage="The Lens Moving Average uses a simple arithmetic mean of the window and applies a skip policy for gaps: 
+            this means that for missing values the bucket is skipped and the calculation is performed on the next one."
+        />
+      </p>
+
+      <p>
+        <FormattedMessage
+          id="xpack.lens.indexPattern.movingAverage.tableExplanation"
+          defaultMessage="For example, given the data [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], we can calculate a simple moving average with windows size of 5 as follows:"
+        />
+      </p>
+
+      <ul>
```

Comment:
I'm happy with merging as is, but this isn't 100% accurate - for the beginning of the series we are working with a partial window. So for [1, 2, 3, 4, 5, 67, 8, 9, 10], you would actually see the following:
* 1 / 1 = 1
* (1 + 2) / 2 = 1.5
* (1 + 2 + 3) / 3 = 2
* (1 + 2 + 3 + 4) / 4 = 2.5
* (1 + 2 + 3 + 4 + 5) / 5 = 3
* (1 + 2 + 3 + 4 + 5 + 6) / 5 = 4
* ...

Reply:
Also there is no moving average for the first position- we should probably show the missing values too.